app/models/dynamic_dialog_field_value_processor.rb

```diff
@@ -12,7 +12,10 @@ def values_from_automate(dialog_field)
     )
 
     dialog_field.normalize_automate_values(workspace.root.attributes)
-  rescue
+  rescue => e
+    $log.error "DynamicDialogFieldValueProcessor: error getting value from automate #{e.message}"
+    $log.error e.backtrace.join("\n")
```

Comment:
Looking into this with @bdunne and there seems to be a problem with `_log` being exposed as an instance method.  It is available if you call the class method like `self.class._log.error <message>`.

We will look into correcting this, but what you have is good for now.